draft-ietf-trans-rfc6962-bis.md

```diff
@@ -950,10 +951,13 @@ extension that it does not understand, it SHOULD ignore that extension.
 Furthermore, an implementation MAY choose to ignore any extension(s) that it
 does understand.
 
-The encoding of the digitally-signed element is defined in [RFC5246].
+`algorithm` indicates the signature scheme used to compute the signature.  The
+SignatureScheme enumeration is defined in [I-D.ietf-tls-tls13].
 
-`timestamped_entry` is a `TransItem` structure that MUST be of type
-`x509_entry_v2` or `precert_entry_v2` (see {{tree_leaves}}).
+`signature` is the value of a signature computed using the appropriate signature
+algorithm for the log indicated by `log_id`.  The input to the signature is a
```

Comment:
Actually, I suggest applying "s/signing algorithm/signature algorithm/" to my previous comment and to the Log Parameters section.